src/datachain/cli.py

```diff
@@ -909,8 +935,28 @@ def rm_dataset(
     name: str,
     version: Optional[int] = None,
     force: Optional[bool] = False,
+    studio: bool = False,
+    team: Optional[str] = None,
 ):
-    catalog.remove_dataset(name, version=version, force=force)
+    if studio:
+        remove_studio_dataset(team, name, version, force)
+    else:
+        catalog.remove_dataset(name, version=version, force=force)
+
+
+def edit_dataset(
+    catalog: "Catalog",
+    name: str,
+    new_name: Optional[str] = None,
+    description: Optional[str] = None,
+    labels: Optional[list[str]] = None,
+    studio: bool = False,
+    team: Optional[str] = None,
+):
+    if studio:
```

Comment:
[Q] Is there a reason for deviating from the behaviour implemented in #561? Why can't a user edit/remove a dataset from both local and remote in the same call?

Reply:
For list operation, having the option to list the datasets from both local and remote seems like a frequent operation. 

But I am not sure how helpful it will edit or remove dataset from both local and remote in the same call.

 I was also looking at the implementation of `pull` where `ds://` prefix is used to identify if it is remote or if it is local. I don't know if it will be confusing to user though to implement it in edit and remove as well. cc. @shcheklein

Reply:
> But I am not sure how helpful it will edit or remove dataset from both local and remote in the same call.

If you think about this in terms of https://github.com/iterative/datachain/issues/578 users would definitely want to be able to rename both at the same time. We want to avoid users ending up with datasets with the same identifier being named differently between local and remote as much as possible.

Reply:
Addressed,.